src/pages/home/HeaderView.js

```diff
@@ -83,6 +83,9 @@ const HeaderView = (props) => {
                                 if (participants.length === 1) {
                                     Navigation.navigate(ROUTES.getDetailsRoute(participants[0]));
                                 }
+                                if (participants.length > 1) {
```

Comment:
Rather than check if `participants.length > 1` we can just return early when the `length === 1` then in all other cases navigate to the new view.